### ISSUE TITLE
[WORDPRESS] Rights issue - impossible to update or install plugins after an upgrade of WP image version > 5.7.1-debian-10-r20

### ISSUE THREAD

**Issue by corend** [bitnami/charts#6407]
**Which chart**:
Wordpress version superior to 5.7.1-debian-10-r20

**Describe the bug**
If you run the Wordpress HELM upgrade from a version below or equal to 5.7.1-debian-10-r20 to upgrade it to a version superior, the installation or update of any plugins are not possible anymore due to rights lost.

**To Reproduce**

1. Install Wordpress version 5.7.1-debian-10-**r5** or below via Bitnami HELM chart, the installation I made is running since weeks on a RWX setup
```YAML
image:
  registry: docker.io
  repository: bitnami/wordpress
  tag: 5.7.1-debian-10-r5
```
2. Update a plugin or install a new one --> **SUCCESS**
3. Upgrade your WP version to 5.7.1-debian-10-**r20** 
```YAML
image:
  registry: docker.io
  repository: bitnami/wordpress
  tag: 5.7.1-debian-10-r20
```
4. Update a plugin or install a new one --> **SUCCESS**
5. Upgrade your WP version to 5.7.1-debian-10-**r21** or upper than that (I also tried with version 5.7.2-debian-10-r4 which is currently the latest)
```YAML
image:
  registry: docker.io
  repository: bitnami/wordpress
  tag: 5.7.1-debian-10-r21
```
OR
```YAML
image:
  registry: docker.io
  repository: bitnami/wordpress
  tag: 5.7.2-debian-10-r4
```
6. Update a plugin or install a new one --> **ERROR**
7. See error: 
`[php7:warn] [pid 146] [client IP:PORT] PHP Warning: unlink(/opt/bitnami/wordpress/tmp/disable-emojis.1.7.2-tLdgC3.tmp): No such file or directory in /opt/bitnami/wordpress/wp-admin/includes/file.php on line 1092, referer: https://domain.name/wp-admin/plugin-install.php?s=disable+emojis&tab=search&type=term`
8. Same issue with all versions from 5.7.1-debian-10-**r21** to 5.7.2-debian-10-r4

**How did I execute the WP upgrade?** 
Simply by changing the image tag to the appropriate version.
```YAML
image:
  registry: docker.io
  repository: bitnami/wordpress
  tag: 5.7.1-debian-10-r20
```

**Why did I do all those version tests?** 
I would like to update WP to the latest version (5.7.2-debian-10-r4), before I was running in version 5.7.1-debian-10-r5, so I tried to identified since which version of the image we are facing this issue. Following my tests, this issue appeared in the version 5.7.1-debian-10-r21.

**Expected behavior**
Should be able to upgrade the WP image without loosing any user rights on the different folders or if new folders are created it should be automatically properly assigned to the correct user and rights.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
Version:"v3.5.4"
```

- Output of `kubectl version`:

```
v1.21.0
```

I suspect then the issue is there since those changes in version 5.7.1-debian-10-r21, I see a lot of changes regarding CHMOD rights:
https://github.com/bitnami/bitnami-docker-wordpress/compare/5.7.1-debian-10-r20...5.7.1-debian-10-r21

**Comment by javsalgar** [comment]
Hi,

I was able to reproduce the issue. A workaround to fix it is to enter the container and execute

```
mkdir /opt/bitnami/wordpress/tmp
```

We will work on fixing it.

**Comment by corend** [comment]
Thank you @javsalgar this is much appreciated.

M2c: I think that in the testing steps we should detect this kind of regression like doing an end 2 end test installing a plugin and a theme to see if we are still able to do it. What do you think? And also in different modes like RWO, RWX...

**Comment by javsalgar** [comment]
Hi,

We normally do this kind of tests, but inside a major version. In this case, when transitioning the conainer to bash `-r21`, we bumped the major version of the chart. Therefore, our automated system is not able to detect this situation. However, we did perform several manual tests so we may have missed something in this case.

**Comment by javsalgar** [comment]
Hi,

Could you try with the latest version of the chart? We just released a container `5.7.2-debian-10-r6` that has the tmp folder with the proper permissions.

**Comment by corend** [comment]
Hi @javsalgar ,

I confirm, this is working as expected with the version 5.7.2-debian-10-r6. Thank you!
